src/app/core/services/local-storage.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+
+import { Videos } from '@core/models/video.model';
+
+@Injectable({
+	providedIn: 'root'
+})
+export class LocalStorageService {
+
+	public saveToLocalStorage(userVideosList: Videos){
+		localStorage.setItem("video-app", JSON.stringify(userVideosList));
```

Comment:
video-app  to pasuje gdzieś przypisać jeśli korzystasz z tego w kilku miejscach to do evironments ale jeśli tylko tutaj to przypisz do jakiejś zmiennej pod deklaracją klasy